e2e/tests/visual-regression.spec.ts

```diff
@@ -24,10 +25,38 @@ test.describe('@visual-regression', () => {
         test('matches the saved screenshot', async ({ page }) => {
           const locator = selector ? page.locator(selector) : page.getByRole('document')
 
-          await expect(locator).toHaveScreenshot([
-            screenshotName,
-            'js-disabled.png',
-          ], { scale: 'css', timeout: 20_000 })
+          const boundingBox = selector ? await locator.boundingBox() : null
+
+          if (boundingBox && boundingBox.height > vMax) {
+            const remainder = boundingBox.height % vMax ? [boundingBox.height % vMax] : []
+
+            const batches = [
+              ...Array.from<number>({ length: Math.floor(boundingBox.height / vMax) }).fill(vMax),
+              ...remainder,
```

Comment:
Looks a bit hard to read any suggestions to simplify it are welcome.
Basically if I want a chunk of 50px and the total is 110px the method should return 
`[50,50,10]`